podaac/subsetter/subset.py

```diff
@@ -965,11 +966,7 @@ def _rename_variables(dataset, base_dataset):
         variable = dataset.variables[var_name]
         var_dims = [x.split(GROUP_DELIM)[-1] for x in dataset.variables[var_name].dims]
         if not var_dims:
-            var_group_parent = var_group
-            # This group doesn't contain dimensions. Look at parent group to find dimensions.
-            while not var_dims:
-                var_group_parent = var_group_parent.parent
-                var_dims = list(var_group_parent.dimensions.keys())
+            var_dims = []
```

Comment:
Wouldn't `var_dims` already be an empty list? It seems like this line does nothing. Also I'm a little concerned about this breaking functionality with our existing datasets

Reply:
Yes, I removed the var_dims reassign. Do you have an idea of what existing datasets might break because all the unittests pass?